apps/api/src/front.js

```diff
@@ -21,6 +23,21 @@ const initContextState = ctx => {
     ctx.state.internalError = false;
     ctx.state.request = {};
     ctx.state.success = false;
+    ctx.state.stats = {};
+};
+
+const getStats = async ctx => {
+    if ("undefined" === typeof ctx.state.stats) {
```

Comment:
That can't happen with the initContactState on top, isn't it?

Reply:
That shoudn't happen, but it could as the 2 func are unrelated, i think... But i will fix that anyway as this function will only return stats object.